main/lib/src/RunControl.cc

```diff
@@ -80,8 +82,12 @@ namespace eudaq {
     SendCommand("CONFIG", to_string(config));
     if (config.SetSection("RunControl")) {
       m_runsizelimit = config.Get("RunSizeLimit", 0LL);
+      m_runeventlimit = config.Get("RunEventLimit", 0LL);
+      m_nextconfigonfilelimit = config.Get("NextConfigFileOnFileLimit", false);
```

Comment:
afaik @RPeschke uses the feature of the run control being scriptable and does not rely on above mentioned feature - but correct me if I'm wrong.